File: Assets/Resources/Scripts/Delay effect.js
#pragma strict

var amount : float = 0.03;
var maxAmount : float = 0.04;
var smooth : float = 2;
private var def : Vector3;
 
function Start (){
    def = transform.localPosition;
}
 
function Update (){
 
    var factorX : float = -Input.GetAxis("Mouse X") * amount;
	var factorY : float = -Input.GetAxis("Mouse Y") * amount;
   
	factorX = Mathf.Clamp(factorX, -maxAmount, maxAmount);
	factorY = Mathf.Clamp(factorY, -maxAmount, maxAmount);

	var Final : Vector3 = new Vector3(def.x+factorX, def.y+factorY, def.z);
	transform.localPosition = Vector3.Lerp(transform.localPosition, Final, Time.deltaTime * smooth);       
}

File: Assets/Scripts/ArrowBow/DelayEffect.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class DelayEffect : MonoBehaviour {

    public float amount = 0.03f;
    public float maxAmount = 0.04f;
    public float smooth = 2;
    private Vector3 def;

	void Start () {
        def = transform.localPosition;
    }
	
	void Update () {
        float factorX = -Input.GetAxis("Mouse X") * amount;
        float factorY = -Input.GetAxis("Mouse Y") * amount;

        factorX = Mathf.Clamp(factorX, -maxAmount, maxAmount);
        factorY = Mathf.Clamp(factorY, -maxAmount, maxAmount);

        Vector3 Final = new Vector3(def.x + factorX, def.y + factorY, def.z);
        transform.localPosition = Vector3.Lerp(transform.localPosition, Final, Time.deltaTime * smooth);
    }
}
